### PR TITLE
feature/rest-api

### PR DESCRIPTION
**Pull request by CodyBrewer**
# Description

We are replacing the graphql api with a REST API. 
We will not be using grahql-yoga to create our server, we will use express.

The express server should run with a default port or a port set as an environment variable.

The express server should have third-party middleware:
- helmet
- express.json
- cors

GET http://HOST:PORT/api/ should return 200 OK with 
```json
{ "status": "up" }
```

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [x] Incomplete/work-in-progress, PR is for discussion/feedback

## How Has This Been Tested?

- [x] Unit test written for /api/ route
- [x] Server ran locally, manually tested /api/ route.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by CodyBrewer** [comment]
[Set up express server](https://trello.com/c/TaMPrQDV/96-set-up-express-server)

**Comment by CodyBrewer** [comment]
📖 self-review complete 📘

**Review by dave-vazquez — APPROVED** [review]
Love the emoji commits! 😆  Let's get this re-write started!! 🎉

File: tests/server.spec.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const request = require('supertest');
+const server = require('../api/server.js');
+
+describe('server', () => {
```
Comment (dave-vazquez) [inline tests/server.spec.js:4]: Unit tests 🏆

**Review by dave-vazquez — APPROVED** [review]

**Pull request merged**